server/services/gamecontrol.go

```diff
@@ -12,19 +13,20 @@ type GameControlServer struct {
 	pb.UnimplementedGameControlServiceServer
 }
 
-func convertGameStartRequestInward(req *pb.GameStartRequest) controller.GameStartRequest {
-	domGameStartRequest := controller.GameStartRequest{}
+func convertGameStartRequestInward(req *pb.GameStartRequest) d.GameStartRequest {
+	domGameStartRequest := d.GameStartRequest{}
```

Comment:
I personally prefer a name that is harder to confuse with "DOM."  Like `domain` as a prefix instead.

Reply:
I refactored this prefix on this advice.